src/components/Home.js

```diff
@@ -138,7 +138,7 @@ export default function Home(props) {
                   },
                 }}
               >
-                Learn more
+                Read about Treetracker
```

Comment:
I think "Read More About Treetracker" or "Read More About The Treetracker System" would be a bit better. Its also better to capitalize the first letter of each word when it comes to seo. What do you think of those phrases @dadiorchen? This is a great step in the right direction though!